### ISSUE TITLE
Lyric Scrapper

### ISSUE THREAD

**Issue by EricStukenberg** [KEXPKaraoke/backend#3]
Input
- artist, song title
Outputs 
- Lyrics 

https://lyrics.fandom.com/wiki/Category:Artists_B

**Comment by sherlocktech** [comment]
Just wanted to double check about the version of Python you're planning on using here. I feel like we should just the up to date version 3.8. Can you check what version you have currently?

**Comment by EricStukenberg** [comment]
Yeah, let's just use 3.8.5 I just updated it now, we should be good.

**Comment by EricStukenberg** [comment]
Curious about what we want the scraper output to be? Right now it should be raw HTML but I can convert that to whatever in the scraper or we can do that later.

**Comment by sherlocktech** [comment]
I’m thinking we should have the scrapers return JSON. We can talk about this more tomorrow though. I missed this comment, sorry for the late reply. From: EricStukenbergSent: Wednesday, August 19, 2020 3:40 PMTo: KEXPKaraoke/backendCc: Sean Stamm; CommentSubject: Re: [KEXPKaraoke/backend] Lyric Scrapper (#3) Curious about what we want the scraper output to be? Right now it should be raw HTML but I can convert that to whatever in the scraper or we can do that later.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
